Training/1.0 Introduction.md

```diff
@@ -12,37 +12,53 @@ First we we learn a little bit about coding in Python, although many of you will
 List all the Data Types of Python with examples.  
 
 ```
-
+list = ["one",2,"3"]
+string = "Hello World!"
+integer = 1024
+boolean = True
+dictionary = {"one":1,"two":2}
+tuple = (1,2,3,4)
 ```
 
 ### Task 2: 
 
 Other useful ways of storing data are in List, Arrays and Tuples. Create one of each and assign them to the variables a, b and c respectively. 
 
 ```
-
+a = ["I", "am", "a", "list"]
+b = [1,2,3,4]
+c = (1,2,3,4)
 ```
 
 ### Task 3: 
 
-We are also interested in becoming good software developers so we will need to use conditional, loops. Write an statement where the summation of a variable x+1 will be calculated if the value of x is greater than 2. 
+We are also interested in becoming good software developers so we will need to use conditional statements. Write a statement where the summation of a variable x+1 will be calculated if the value of x is greater than 2. 
 
 ```
-
+x = 5
+if x > 2:
+  print(x++)
 ```
 
-Now incorporate a for loop to calculate the value of n(x+1) for n repetitions where n=3. Store each value of the for loop in an array named y.  
+Now incorporate a for loop to calculate the value of n(x+i) for n repetitions where n=3. Store each value of the for loop in an array named y.  
 
 ```
-
+y = []
+for i in range(n):
+  y.append(n*(x+i))
```

Comment:
Correct use of a for loop, but you've altered the task - the prompt said add +1 to x not +i. To make and append an array, you have to use numpy. Proper solution would be:
`import numpy`
`y = numpy.array([])`
`for i in range(n):`
 `   y = numpy.append(y,n*(x+1))`